standalone-integration/src/test/java/com/ericsson/bss/cassandra/ecchronos/standalone/ITTableRepairJob.java

```diff
@@ -98,14 +100,22 @@
     }
 
     @Parameterized.Parameters
-    public static List<RepairHistoryType> repairHistoryTypes()
+    public static Collection repairHistoryTypes()
     {
-        return Arrays.asList(RepairHistoryType.values());
+        return Arrays.asList(new Object[][] {
+                {RepairHistoryType.CASSANDRA, true},
+                {RepairHistoryType.CASSANDRA, false},
+                {RepairHistoryType.ECC, true},
+                {RepairHistoryType.ECC, false}
+                });
     }
 
     @Parameterized.Parameter
     public RepairHistoryType myRepairHistoryType;
 
+    @Parameterized.Parameter(1)
+    public Boolean myRemoteRoutingOption;
```

Comment:
Not used anywhere. Is this needed for the Parameterized tests?

Reply:
They arw needed for the parameterized things to work.